transports/httpclient/src/main/java/org/commonjava/maven/galley/transport/htcli/HttpClientTransport.java

```diff
@@ -122,7 +122,7 @@ public DownloadJob createDownloadJob( final ConcreteResource resource, final Tra
     {
         return new HttpDownload( getUrl( resource ), getHttpLocation( resource.getLocation(), download ), target,
                                  transferSizes, eventMetadata, http, mapper, metricRegistry, metricConfig,
-                                 globalProxyConfig.getEgressSites(), proxySitesCache );
+                globalProxyConfig == null ? null : globalProxyConfig.getEgressSites(), proxySitesCache );
```

Comment:
If an HttpClientTransport was manually created the globalProxyConfig might be null so this needs a null check. Found this when testing the updated version against PME.